clients/client-python/gravitino/api/expressions/named_reference.py

```diff
@@ -0,0 +1,76 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Why do we call this a named reference?
Is there a case where a reference has no name?
Do we have to differentiate these two types of references?

Reply:
@xunliu  Do you have any idea about the name for this feature?

Reply:
hi @tengqm 
because we need to implement all classes in the Python client.
- https://github.com/apache/gravitino/blob/main/api/src/main/java/org/apache/gravitino/rel/expressions/NamedReference.java

Reply:
@mchades Do you have any suggestions?

Reply:
Really appreciate this clarification. Thanks.